### ISSUE TITLE
Move admin-assigned submissions to "Submissions to Score" table for judges

### ISSUE THREAD

**Issue by shaunxp20** [Iridescent-CM/technovation-app#3869]
When an admin assigns a submission to a judge, and then the judge goes to score this submission:
- it should appear in the "Submissions to Score" table **not** the "Scores in Progress" table
- after the score has been started by the judge, it should move to the "Scores in Progress" table

Discuss:
- For the "Submissions to Score" table, is there any particular order the submissions should be in? (We'll be combining the admin assigned submissions with the RPE/ChA assigned submissions).

Notes: https://github.com/Iridescent-CM/technovation-app/issues/3799#issuecomment-1440908255

## BASIC QA NOTES

Both an RPE judge and non-RPE judge should be tested to ensure scores are displaying correctly in their "Submissions to Score" table and their "Scores in Progress" table, and for quarterfinals and semifinals judging rounds.

**Comment by dboyer** [comment]
I think it might be nice to have RPE assigned submissions grouped together and admin assigned submissions together but don't think there needs to be any particular order other than that.

**Comment by shaunxp20** [comment]
This one is on QA, I added some basic QA notes too.

**Comment by dboyer** [comment]
Confirmed for RPE judges that assigned projects appear in the submissions to score and move to the in progress and finished sections as expected.

Confirmed for online judges that assigned projects appear in the submissions to score and move to the in progress and finished sections as expected.

I also tested for someone who judged at an RPE and then I manually assigned them an online score and all worked as expected.
